Refit/targets/refit.props

```diff
@@ -4,4 +4,17 @@
     <CompilerVisibleProperty Include="RefitInternalNamespace" />
   </ItemGroup>
 
+  <Choose>
+    <When Condition="'$(VisualStudioVersion)' == '17.0' AND '$(DesignTimeBuild)' == 'True'">
```

Comment:
Should this (logically) be:

```
if '$(VisualStudioVersion)' >= '17.0'
```

? I know it will be a while until we have `18` or `17.1`, but it might make sense to guard against that now.

Reply:
Historically, the minor version doesn't change here, so the next change would be `18.0`. Prior to that, we're going to need to address the design time build limitation.

With that said, if MSBuild supports `>=` here it doesn't hurt.